<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OscilloscopeWindow</class>
 <widget class="QDialog" name="OscilloscopeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>88</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audio Oszilloskop</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="OscilloscopeWidget" name="oscilloscope0" native="true"/>
   </item>
   <item>
    <widget class="OscilloscopeWidget" name="oscilloscope1" native="true"/>
   </item>
   <item>
    <widget class="OscilloscopeWidget" name="oscilloscope2" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OscilloscopeWidget</class>
   <extends>QWidget</extends>
   <header>oscilloscope_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
